docs_user/modules/con_ceph-daemon-cardinality.adoc

```diff
@@ -0,0 +1,39 @@
+[id="ceph-daemon-cardinality_{context}"]
+
+= {Ceph} daemon cardinality
+
+{Ceph} 6 and later applies strict constraints in the way daemons can be colocated within the same node.or more information, see link:https://access.redhat.com/articles/1548993[Red Hat Ceph Storage: Supported configurations]. The resulting topology depends on the available hardware, as well as the amount of Red Hat Ceph Storage services present in the Controller nodes which are going to be retired. The following document describes the procedure required to migrate the RGW component (and keep an HA model using the https://docs.ceph.com/en/latest/cephadm/services/rgw/#high-availability-service-for-rgw[Ceph Ingress daemon] in a common {OpenStackPreviousInstaller} scenario where Controller nodes represent the
+https://github.com/openstack/tripleo-ansible/blob/master/tripleo_ansible/roles/tripleo_cephadm/tasks/rgw.yaml#L26-L30[spec placement] where the service is deployed. As a general rule, the
+number of services that can be migrated depends on the number of available nodes in the cluster. The following diagrams cover the distribution of the {Ceph} daemons on the {Ceph} nodes where at least three nodes are required in a scenario that sees only RGW and RBD, without the {dashboard_first_ref}:
+//kgilliga: We probably shouldn't link to the opensource docs or to github in the downstream docs. I need to find a downstream equivalent or we should remove these links.
```

Comment:
We can refer to this: https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/6/html-single/object_gateway_guide/index#high-availability-for-the-ceph-object-gateway
that is supposed to describe the same thing downstream.

Reply:
Also, instead of linking to github we can just remove the link downstream.